service/src/main/java/org/cbioportal/genome_nexus/service/internal/BaseVariantAnnotationServiceImpl.java

```diff
@@ -229,7 +229,7 @@ private VariantAnnotation getVariantAnnotation(String variant, EnrichmentService
                 postEnrichmentService.enrichAnnotations(variantAnnotations);
             }
         } catch (VariantAnnotationWebServiceException e) {
-            LOG.warn(e.getLocalizedMessage());
+            LOG.warn(e.getLocalizedMessage(), e);
```

Comment:
added this here since it was very useful in tracking down the error causing the issue fixed by this pr